recipes/ganon/meta.yaml

```diff
@@ -30,18 +30,20 @@ requirements:
     - catch2 ==2.*
     - cxxopts >=2.2.0
     - python
-    - pandas >=1.1.0
+    - pandas >=1.2.0
     - multitax >=1.3.1
     - genome_updater >=0.6.3
     - grep
     - coreutils
     - curl
     - diffutils
     - zlib
+    - parameterized >=0.9.0
+    - raptor ==3.0.1
```

Comment:
```suggestion
    - raptor >=3
```

Reply:
I know that v4 will introduce breaking changes, so I fixed on `==3.*`, is that ok?